Documentation/git-commit.txt

```diff
@@ -176,7 +176,8 @@ The `-m` option is mutually exclusive with `-c`, `-C`, and `-F`.
 	Usually recording a commit that has the exact same tree as its
 	sole parent commit is a mistake, and the command prevents you
 	from making such a commit.  This option bypasses the safety, and
-	is primarily for use by foreign SCM interface scripts.
+	is primarily for use by foreign SCM interface scripts. See the 
+	`commit.allowEmpty` configuration variable in linkgit:git-config[1].
```

Comment:
The new sentence is slightly wordy - could it be more concise -perhaps
 s/configuration variable //